docs/admin/provisioning_usb.rst

```diff
@@ -2,3 +2,41 @@ Provisioning Export USB devices
 ===============================
 
 .. include:: ../includes/top-warning.rst
+
+SecureDrop Workstation supports the export of submissions from the Qubes client
+to an encrypted USB *Export Device*. 
+
+.. note:: Currently only LUKS-encrypted devices are supported,
+  which effectively restricts the *Export Device* to use with Linux-based
+  systems such as Tails. Support for Veracrypt-encrypted devices is planned,
+  which will allow the use of the *Export Device* with MacOS and Windows systems.
+
+In order to provision an *Export Device* for  use with SecureDrop Workstation,
+you will need a fresh USB stick and a Linux-based system. Tails is recommended -
+if available, the  *Secure Viewing Station* can be used, adding the extra benefit
```

Comment:
another spaces nit between "the" and "*Secure"

Reply:
:heavy_check_mark: